go.mod

```diff
@@ -5,13 +5,13 @@ go 1.17
 require (
 	github.com/aws/aws-lambda-go v1.27.0
 	github.com/aws/aws-sdk-go-v2 v1.16.1
+	github.com/google/go-cmp v0.5.7
```

Comment:
Can it be a dev dependency?

Reply:
this is handled by go build
"When compiling packages, build ignores files that end in '_test.go'."
https://pkg.go.dev/cmd/go#hdr-The_main_module_and_the_build_list